e2e/tests/regression/volume_basic.robot

```diff
@@ -0,0 +1,40 @@
+*** Settings ***
+Documentation    Basic Test Cases
+
+Test Tags    regression
+
+Resource    ../keywords/common.resource
+Resource    ../keywords/deployment.resource
+Resource    ../keywords/persistentvolumeclaim.resource
+Resource    ../keywords/recurringjob.resource
+Resource    ../keywords/statefulset.resource
+Resource    ../keywords/volume.resource
+
+Test Setup    Set test environment
+Test Teardown    Cleanup test resources
+
+*** Variables ***
+${LOOP_COUNT}    1
+${RETRY_COUNT}    300
+${RETRY_INTERVAL}    1
+
+*** Test Cases ***
+Volume Basic Test
+    When Create volume wrong_volume-name-1.0
+    Then No volume created
+
+    When Create volume wrong_volume-name
+    Then No volume created
+
+    When Create volume 0 with frontend invalid_frontend
+    Then No volume created
+
+    When Create volume 0 with 2 GB and 3 replicas
+    And Attach volume 0
+    Then Validate volume 0 replicas anti-affinity
```

Comment:
Do we need to add a check for volume heathy state? If I understang correctly, `validate_volume_replicas_anti_affinity` only check replica counts, if replica not running the volume still can wirte data.

No other questions, thank you.

Reply:
Added. Thank you!